dummy.c

```diff
@@ -0,0 +1 @@
+/* Dummy change - fixed syntax */
```

Comment:
This file has been removed via a rewrite of git history. I think the right way for you to fix this is to `git rebase upstream/BABEL_1_X_DEV`